lib/recurly/usage.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+class Recurly_Usage extends Recurly_Resource
+{
+  var $subUuid;
+  var $addOnCode;
```

Comment:
ugh... these look funny but i guess we use mostly camel case with a little snake case for instance variables... oh well